common/src/main/java/io/homeassistant/companion/android/common/sensors/LastUpdateManager.kt

```diff
@@ -71,9 +71,12 @@ class LastUpdateManager : SensorManager {
         if (settingsToRemove.isNotEmpty()) {
             sensorDao.removeSettings(lastUpdate.id, settingsToRemove.map { it.name })
         }
-        val intentSettings = allSettings.filter {
+        val intentSettingsUnsorted = allSettings.filter {
             it.name.startsWith(INTENT_SETTING_PREFIX)
         }
+        val intentSettings = intentSettingsUnsorted.sortedBy {
```

Comment:
Is there a specific reason you're introducing the `intentSettingsUnsorted` variable, instead of chaining the calls (`allSettings.filter { ... }.sortedBy { ... }`)? As far as I can see it is only used for sorting here.

Reply:
Good point, there's no specific reason. Fixed.